Imagekit.UnitTests/ImageKitTestCasesNonAsync.cs

```diff
@@ -0,0 +1,694 @@
+using Imagekit.Constant;
+using Imagekit.Sdk;
+using Imagekit.Models;
```

Comment:
Generic feedback for all test cases as discussed before and shared in the loom video.

## Basic error handling
* SDK is throwing an error if required parameters are missing. No need to do type checking separately.

## HTTP Request body assertions
We need to assert the following in all API methods:
* Request body is correct with full-fledged example with all properties set.
* HTTP endpoint is correct.
* HTTP method is correct.
* Basic authentication parameters are correct.

## Response body assertion
We will mock the success and error response instead of calling the imagekit endpoint. 
* Success body is set
* Error body is set
* ResponseMetadata is correctly set